### PR TITLE
Improve logging

### PR DESCRIPTION
**Pull request by benrubson**
Hello,

This PR does 2 things :
- it removes timestamp from log lines when syslog in use (solves #270) ;
- it adds `-t` option to tune syslog tag.

Thank you 👍 

Ben

### CODE REVIEW

**Comment by benrubson** [comment]
@rfjakob could we think about merging this easy PR please ? Many thx 👍

File: encfs/main.cpp [hunk 1]
```diff
@@ -190,6 +191,7 @@ static bool processArgs(int argc, char *argv[],
   out->isVerbose = false;
   out->idleTimeout = 0;
   out->fuseArgc = 0;
+  out->syslogTag.assign("encfs");
```
Comment (rfjakob) [inline encfs/main.cpp]: Maybe a stupid question, but why not simply

    out->syslogTag = "encfs";

File: encfs/main.cpp [hunk 2]
```diff
@@ -265,6 +269,9 @@ static bool processArgs(int argc, char *argv[],
       case 'S':
         out->opts->useStdin = true;
         break;
+      case 't':
+        out->syslogTag.assign(optarg);
```
Comment (rfjakob) [inline encfs/main.cpp]: Why not use "="?

File: encfs/main.cpp (hunk 1 above)
Comment (benrubson) [inline encfs/main.cpp]: Because option `p)` some lines below uses `assign` :)
I updated with `=` which is more convenient 👍

**Comment by dickerpulli** [comment]
When is 1.9.2 going to be released?

**Comment by benrubson** [comment]
@rfjakob / @vgough will decide, but would be nice to try to solve #214 / #244 I think.
And merge #293 / #295 / #309 / #317.
Here's my wishlist for 1.9.2 :)

**Pull request merged**